src/renderer/components/AssetsFilter/AssetsFilter.tsx

```diff
@@ -27,7 +27,7 @@ export const AssetsFilter: React.FC<Props> = ({ poolFilters, className, activeFi
       __base__: intl.formatMessage({ id: 'common.asset.base' }),
       __usd__: 'usd',
       __erc20__: 'erc20',
-      __bep2__: 'bep2',
+      __avax__: 'avax',
```

Comment:
I did this too but this filter would only show usd as there are only two avax assets listed.

Reply:
thats okay